src/style.css

```diff
@@ -0,0 +1,101 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Awenir" ;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Awenir" ;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+body {
+  font-family: "Awenir", sans-serif;
+  font-weight: 300;
+}
+
+.search-form {
+  border: 0;
+  height: 70px;
+  margin-top: 20px;
+  width: 100%;
+  background: #fff;
+}
+
+.input {
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+  height: 100%;
+  width: 100%;
+  font-weight: 800;
+  color: #3d4e61;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-family: inherit;
+  outline: none;
+}
+
+.search-form-small {
+  height: 42px;
+}
+
+.image {
+  display: block;
```

Comment:
It's unnecessary rule

Reply:
![icon-display](https://user-images.githubusercontent.com/120708868/214507148-ab3659d3-e022-4e37-ac50-f12a0269fabc.PNG)
I agree with you, i just copied from photo.